src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKey);
+  }
+
+  handleKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">{ pressedKey !== '' ? `The last pressed key is [${pressedKey}]` : 'Nothing was pressed yet'}</p>
```

Comment:
also you can do without the expression " pressedKey !== '' "
```suggestion
        <p className="App__message">{ pressedKey 
          ? `The last pressed key is [${pressedKey}]`
           : 'Nothing was pressed yet'}
         </p>
```

Reply:
Agree, do it